### PR TITLE
'Solution'

### PR DESCRIPTION
**Pull request by LaskoA**
Please check my solution, thank you :

)

### CODE REVIEW

File: taxi/urls.py [hunk 1]
```diff
@@ -1,9 +1,21 @@
 from django.urls import path
 
-from .views import index
+from .views import index, ManufacturerListView, CarListView, CarDetailView,\
+    DriverListView, DriverDetailView
```
Comment (Abnormaltype) [inline taxi/urls.py:4]:
```suggestion
from .views import
    index, 
    ManufacturerListView, 
    CarListView, 
    CarDetailView,
    DriverListView, 
    DriverDetailView
```

**Review by Abnormaltype — CHANGES_REQUESTED** [review]

**Comment by LaskoA** [comment]
Will it be OK now?

File: taxi/urls.py [hunk 2]
```diff
@@ -1,9 +1,26 @@
 from django.urls import path
 
-from .views import index
+from .views import \
+    index, \
+    ManufacturerListView, \
+    CarListView, \
+    CarDetailView, \
+    DriverListView, \
+    DriverDetailView
```
Comment (Abnormaltype) [inline taxi/urls.py:9]:
```suggestion
from .views import (
    index, 
    ManufacturerListView, 
    CarListView, 
    CarDetailView, 
    DriverListView, 
    DriverDetailView,
)
```

**Review by Abnormaltype — CHANGES_REQUESTED** [review]
😉

**Comment by LaskoA** [comment]
added brackets ;)

**Review by Abnormaltype — APPROVED** [review]
Good job!)

**Pull request closed without merging**